integration-tests/src/test/java/org/zowe/apiml/integration/proxy/MultipartPutIntegrationTest.java

```diff
@@ -40,8 +39,7 @@ static void beforeClass() {
 
     protected static URI[] discoverableClientUrls() {
         return new URI[]{
-            HttpRequestUtils.getUriFromGateway(MULTIPART_PATH),
-            HttpRequestUtils.getUriFromGateway(MULTIPART_PATH_OLD_FORMAT)
+            HttpRequestUtils.getUriFromGateway(DISCOVERABLE_MULTIPART)
```

Comment:
Does it make sense to have it parametrized?

Reply:
Removed